assets/css/elementor-frontend-forms.css

```
@@ -0,0 +1,197 @@
+.wpuf-elementor-widget-wrapper.wpuf-elementor-upload-full .wpuf-attachment-upload-filelist .file-selector {
+  width: 100%;
+  display: block;
+}
+.wpuf-elementor-widget-wrapper.wpuf-elementor-hide-labels .wpuf-label {
+  display: none !important;
+}
+.wpuf-elementor-widget-wrapper.wpuf-elementor-hide-errors .wpuf-error,
+.wpuf-elementor-widget-wrapper.wpuf-elementor-hide-errors .error {
+  display: none;
+}
+.wpuf-elementor-widget-wrapper.wpuf-elementor-hide-placeholder input::-webkit-input-placeholder,
```

Stylelint (17.2.0)

[error] Unexpected vendor-prefixed selector "::-webkit-input-placeholder" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)